lib/cli.js

```diff
@@ -38,7 +38,16 @@ var processArgs = function (argv, options, fs, path) {
   }
 
   if (helper.isString(options.logLevel)) {
-    options.logLevel = constant['LOG_' + options.logLevel.toUpperCase()] || constant.LOG_DISABLE
+    var logConstant = constant['LOG_' + options.logLevel.toUpperCase()];
+    if (helper.isDefined(logConstant)) {
+      options.logLevel = logConstant;
+    } else {
+      console.error('Log level must be one of disable, error, warn, info, or debug.');
+      process.exit(1);
+    }
+  } else if (helper.isDefined(options.logLevel)) {
+    console.error('Log level must be one of disable, error, warn, info, or debug.');
+    process.exit(1);
```

Comment:
Actually, if `logLevel` is not defined (i.e. `options.logLevel` is not defined), this is valid. It just means the user did not pass a `--log-level` argument.